bootstrap/004_labels.sql

```diff
@@ -134,6 +134,102 @@ BEGIN
 
 END;
 
+CREATE OR REPLACE PROCEDURE INTERNAL.UPSERT_PREDEFINED_LABEL(name text, grp text, rank number, condition text)
+    RETURNS text
+    LANGUAGE SQL
+    EXECUTE AS OWNER
+AS
+BEGIN
+    if (:name is null) then
```

Comment:
I don't think we need to do validation for predefined labels. Its not user input and we cant do anything if it fails, installation or upgrade would fail.

Instead we should just test (in `test_label_procs.py` ) that the conditions are valid at commit time (the only time we can actually do anything about the conditions being bad).